Sources/Sentry/SentryLevelHelper.m

```diff
@@ -1,8 +1,9 @@
 #import "SentryLevelHelper.h"
 #import "SentryBreadcrumb+Private.h"
 
-NSUInteger
-sentry_breadcrumbLevel(SentryBreadcrumb *breadcrumb)
+@implementation SentryLevelBridge : NSObject
++ (NSUInteger)breadcrumbLevel:(SentryBreadcrumb *)breadcrumb
```

Comment:
`m`: I'm curious about why this change is required.

Reply:
With the new visionOS configuration, we can't direct call C functions from Swift anymore.

Reply:
It would be great to document this in https://github.com/getsentry/sentry-cocoa/blob/main/develop-docs/DECISIONS.md